README.md

```diff
@@ -154,7 +154,23 @@ configured to the Accumulo table where you want phrases to be exported.
 
 Now initialize and start Fluo as outlined in its docs. Once started the
 load and print commands above can be run passing in
-`$FLUO_HOME/conf/fluo.properties`
+`$FLUO_HOME/apps/<appname>/conf/fluo.properties`
+
+There are two example bash scripts that run this test using the Fluo tar
+distribution.  The previous maven commands using the exec plugin are convenient
+for a development environment, using the following scripts shows how things
+would work in a production environment.
+
+  * [run-mini.sh](bin/run-mini.sh) : Runs this example using mini fluo started
+    using the tar distribution.  Running this way does not require setting up
+    Hadoop, Zookeeper, and Accumulo separately.  Just download or build the
+    Fluo tar distribution, untar it, and point the script to that directory.
+    This is the easiest way to simulate a production environment.
+  * [run-cluster.sh] (bin/run-cluster.sh) : Runs this example with YARN using
```

Comment:
I was able to run this script sucessfully.  However, it might be helpful to show the basic usage:
```
./bin/run-cluster.sh <FLUO HOME> <PHRASECOUNT_HOME> <TXT FILES DIR>
```

Also, could be helpfult to mention that only files with `.txt` suffix will be used in `<TXT FILES DIR>` as I tried to specify a directory with markdown `.md` files and no data was loaded.